docs: camera-enablement (#20175)
OpenTrons/opentrons-api: docs/flex/docs/opentrons-app.md

| **Temperature**   | <ul><li>Set module temperature / Deactivate module</li></ul>                            |
| **Thermocycler**  | <ul><li>Set lid temperature / Deactivate lid</li><li>Open lid / Close lid</li><li>Set block temperature / Deactivate block</li></ul> |

## Camera features and controls

> Every Flex comes equipped with a built-in 2.1 megapixel camera that can capture full HD still images and provides video of the deck and working area. Disabled until the release of robot software version 8.8, the camera is now enabled for control and  use in the Opentrons App and from the touchscreen. When on, the camera provides:

Reviewer: I think this is too negative, even if this has been a historical gap.
```suggestion
Every Flex comes equipped with a built-in 2.1 megapixel camera that can capture full HD still images and provides video of the deck and working area. Starting in robot software version 8.8, you can control the camera in the Opentrons App and from the touchscreen. When enabled, the camera provides:
```

Author: Thank you. Applying suggested change.

- Live, in-app monitoring during protocol runs.
- Automatic image capture at protocol-defined intervals.
- Automatic image capture in response to a crash or runtime error.
- The ability to download all still images in a single, compressed file (`.zip` format) after a protocol run.

### Location

The camera is mounted on the interior frame of the robot, in the upper corner of the enclosure near the front door.

![Image showing robot front and interior camera](./images/camera-location.png)

This fixed location gives you a wide-angle field of view of attached instruments, deck-mounted modules, and labware.

> ![Default camera view](./images/camera-default.png) 

Reviewer: This section seems better suited to the system description section. Let's move it there to keep this page focused on app features.

Author: Sure, will move it!

### Controls

The camera is turned off by default. To turn on the camera and access its features:

<div class="instruction-list" markdown>

1. From the Opentrons App, click **Devices** and locate your robot.
2. For your selected robot, click the three-dot menu ( ⋮ ) and then click **Robot settings**.
3. Click the **Camera** tab to open the camera settings.
4. Click the Camera slider to enable or disable the camera.

</div>

> You can also see the camera's status, and turn it on and off, from the Peripherals section of the details page for your Flex. Click the three-dot menu ( ⋮ ) to enable or disable the camera. This example shows a Flex with its camera enabled.

Reviewer: We now have a mix of `( ⋮ )` and `(⋮)` in the manual. We should pick which one we think looks better and standardize (in a separate PR).

Author: Revising to use (⋮)

!!! note

>     If you need to maintain a comprehensive record of all runs performed on your Flex, you must use the **Download run log** feature to save this information to your computer.

Reviewer: It's strange that this note breaks up the list here. Either:
- Put it after the entire list (most like previous revision).
- Put it inside the `Download run log` bullet (indented so it's contained in that list item).

Author: This is maybe for the other PRs that got merged together, not for the camera PR?

> - **Download image files:** Save to your computer a ZIP file containing all the still images taken during a protocol run, if the camera was enabled.

Reviewer: Apparently [nobody can decide](https://books.google.com/ngrams/graph?content=zip+file&year_start=1990&year_end=2022&corpus=en&smoothing=3&case_insensitive=true) whether it's "ZIP" or "zip". Even [the Wikipedia article](https://en.wikipedia.org/wiki/ZIP_(file_format)) mixes usage. We have cleverly avoided the problem in the rest of the manual by using the file extension, and we can do that again here.

```suggestion
- **Download image files:** Save to your computer a `.zip` file containing all the still images taken during a protocol run, if the camera was enabled.
```

Author: Applying suggestion.
